pkg/pligos/config.go

```
@@ -105,10 +104,25 @@ func CreateSchema(schemaPath string, types map[string]interface{}) (map[string]i
 		return nil, err
 	}
 
-	return (&Normalizer{}).Normalize(mergeMaps(types, schema)), nil
+	return (&maputil.Normalizer{}).Normalize(mergeMaps(types, schema)), nil
 
 }
 
+func OpenPligosConfig(path string) (PligosConfig, error) {
+	buf, err := ioutil.ReadFile(filepath.Join(path, "pligos.yaml"))
+	if err != nil {
+		return PligosConfig{}, err
+	}
+
+	var res PligosConfig
+	if err := yaml.Unmarshal(buf, &res); err != nil {
+		return PligosConfig{}, nil
```

Shouldn't we return an error if the `yaml` file is malformed?

---

https://github.com/real-digital/pligos/issues/4